themes/bootstrap3/templates/Helpers/copy-to-clipboard-button.phtml

```diff
@@ -0,0 +1,32 @@
+<?php
+  $buttonNumber = $buttonNumber ?? 1;
```

Comment:
This looks good: only recommendation is this: if we do want a fallback here just in case, it should be a random number or a timestamp instead of 1 to avoid collision.

Reply:
Good idea. I think `random_bytes(5)` would do the trick; I'll give it a try.